shell/platform/windows/win32_window.h

```diff
@@ -106,7 +109,13 @@ class Win32Window {
 
   UINT GetCurrentHeight();
 
+  // Set to true to be notified when the mouse leaves the window.
+  bool tracking_mouse_leave_ = false;
```

Comment:
ivars need to be private, not protected, with accessors if necessary. In this case, rather than an accessor you can just put the toggling of the variable in the superclass. That would make a lot more sense anyway, because what it's tracking is whether `TrackMouseEvent` has fired, and that's superclass logic, not subclass logic.

Reply:
Done